tests/charts/airflow_core/test_worker.py

```diff
@@ -82,6 +82,18 @@ def test_should_add_extra_containers(self):
             "image": "test-registry/test-repo:test-tag",
         } == jmespath.search("spec.template.spec.containers[-1]", docs[0])
 
+    def test_disable_wait_for_migration(self):
+        docs = render_chart(
+            values={
+                "workers": {
+                    "waitForMigrations": {"enabled": False},
+                },
+            },
+            show_only=["templates/workers/worker-deployment.yaml"],
+        )
+        actual = jmespath.search("spec.template.spec.initContainers", docs[0])
+        assert actual is None
```

Comment:
If we add a new init container by default to the worker pod, this test will fail automatically, I wonder if we should assert that the  `initContainers` object is None or it's a list but doesn't contain the `wait-for-airflow-migrations` container. WDYT?

Reply:
Yep, I totally agree. I was trying to keep the same pattern in other tests, but yes, this should be made less brittle.  I'll push up a change later tonight. 👍

Reply:
Let's just make this specific one less brittle, then open a separate PR for the others.

Reply:
Ok, I pushed up that change to narrow the scope of the test.